src/tools/pub/pubspec.md

```diff
@@ -398,6 +403,43 @@ the pattern is considered relative to the package's root directory.
   Support for the `false_secrets` field was added in Dart 2.15.
 {{site.alert.end}}
 
+### Screenshots
+
+Packages may showcase their widgets or other visual elements using screenshots.
+
+A package can include up to 10 screenshots. 
+These are specified in the pubspec as a list under the `screenshots` field. 
+Each screenshot is specified by a `description` and a `path`. 
+The `description` is a string of at most 160 characters
+describing the screenshot. 
```

Comment:
**issue:** Revise to remove passive voice and the circular description.
```suggestion
Specify each screenshot with a `description` and a `path`. 
For the `description`, provide a string of no more than 160 characters
that explains what the screenshot depicts. 
```

Reply:
I'm by no means an expert on writing good documentation, text and such... :rofl: 
But isn't it reasonably fair to say that `https://dart.dev/tools/pub/pubspec` is _reference style_ documentation?

It does not outline how you can publish a package or what you should/can do.
But it aims to document how properties in `pubspec.yaml` are interpreted by pub / pub.dev, what they mean. Whether you're looking to write packages or consume packages. My argument is that this is the closest we have to reference style documentation of the `pubspec.yaml` file format.

I think that's also reflected in the sitemap / menu layout:
![image](https://user-images.githubusercontent.com/149732/197514955-4f26fd64-cc78-44bd-84b4-846e23c87899.png)

For documentation on what you as a package author should do. We have "guide"-style documentation in:
https://dart.dev/tools/pub/publishing
(which should also be updated with guidelines for how to use screenshots).

---------------------------------

So maybe, we don't want to write:
> For the `description`, provide a string of no more than 160 characters
> that explains what the screenshot depicts. 

But use a more passive voice?

https://developers.google.com/style/reference-verbs

Reply:
@jonasfj : Great points actually. Maybe more along the lines of:
```
Each screenshot includes one `description` and one `path`. 
The `description` explains what the screenshot depicts in no more than 160 characters. 
```

Reply:
Done, using the last suggestion.